src/main/java/com/zharnasek/alesya/game/RandomActivities.java

```diff
@@ -0,0 +1,19 @@
+package com.zharnasek.alesya.game;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class RandomActivities {
+
+    public String getRandomActivities(Gamer gamer, GameResource gameResources) {
+        Random rand = new Random();
+        ArrayList<Activities> list = new ArrayList<>();
+        for (int i = 0; i < gameResources.getActivities().size() - 1; i++) {
+            if (gameResources.getActivities().get(i).getGender().equals(String.valueOf(gamer.getGender()))) {
```

Comment:
Need to compare enum, string it's bad for this purpose. Why Gender in activities is String?

Reply:
updated